torch/csrc/jit/passes/quantization/helper.cpp

```diff
@@ -137,6 +137,8 @@ std::vector<std::string> _single_input_general_value_aten_funcs = {
     "elu_",
     "leaky_relu",
     "leaky_relu_",
+    "hardsigmoid",
+    "hardsigmoid_",
```

Comment:
this is fixed qparams op, see _fixed_qparams_map, you will need to include this in `getPassThroughInputs` function

Reply:
from looking at `getPassThroughInputs`, seems like this diff would not change the behavior?  I might be misunderstanding your suggestion.

`hardsigmoid` has fixed output qparams and it does change the input values, which is why it seemed it belongs in `_single_input_general_value_aten_funcs`.  I could be missing something.

Reply:
we will need to include a check for fixed input qparam ops, by calling `getFixedQParams` in `getPassThroughInputs`

`hardsigmoid` has fixed qparams, we have a separate category for that, which is defined in `_fixed_qparams_map`

Reply:
I see, looks like there is an assumption in the current code that the ops with fixed params are not included in things like `_single_input_general_shape_call_funcs`.

I'm going to leave this diff at just the bug fix (since hardsigmoid seems to be left over in the wrong category), and we can do the refactor if needed in a separate PR.